src/components/search-package-list-item/search-package-list-item.js

```diff
@@ -89,12 +91,24 @@ function SearchPackageListItem({
         </div>
       </InternalLink>
     );
-}
+};
 
 SearchPackageListItem.propTypes = {
   active: PropTypes.bool,
   headingLevel: PropTypes.string,
-  item: PropTypes.object,
+  item: PropTypes.shape({
+    isSelected: PropTypes.bool,
+    name: PropTypes.string,
+    providerName: PropTypes.string,
+    selectedCount: PropTypes.number,
+    tags: PropTypes.shape({
+      tagList: PropTypes.arrayOf(PropTypes.string),
+    }),
+    titleCount: PropTypes.number,
+    visibilityData: PropTypes.shape({
+      isHidden: PropTypes.bool,
+    }),
+  }),
```

Comment:
```suggestion
  item: PropTypes.shape({
    isSelected: PropTypes.bool.isRequired,
    name: PropTypes.string.isRequired,
    providerName: PropTypes.string,
    selectedCount: PropTypes.number.isRequired,
    tags: PropTypes.shape({
      tagList: PropTypes.arrayOf(PropTypes.string),
    }).isRequired,
    titleCount: PropTypes.number.isRequired,
    visibilityData: PropTypes.shape({
      isHidden: PropTypes.bool.isRequired,
    }).isRequired,
  }).isRequired,
```
Also, `link` property should be required

Reply:
Correct me if I'm wrong please, but how can `item` property be required if we check for its presence on line 33 and if it's not there we just render skeleton instead?

Reply:
hm.. you are right, sorry, do not fix it. But it's strange, in all places where we are call this component we pull this prop and it's look like required. Probably we need to deep inside into this moment, but this is an another story.